hwilib/devices/trezor.py

```diff
@@ -754,6 +787,20 @@ def toggle_passphrase(self) -> bool:
                 print(PIN_MATRIX_DESCRIPTION, file=sys.stderr)
         return True
 
+    @trezor_exception
+    def can_sign_taproot(self) -> bool:
+        """
+        Trezor T supports Taproot in firmware versions greater than (not including) 2.4.2.
+        Trezor One supports Taproot in firmware versions greater than (not including) 1.10.3.
```

Comment:
This is just semantics but this reads and feels much better:

```suggestion
        Trezor T supports Taproot since firmware version 2.4.3.
        Trezor One supports Taproot since firmware version 1.10.4.
```

Reply:
Done